pipeline_dp/combiners.py

```diff
@@ -25,3 +30,49 @@ def merge_accumulators(self, accumulator1, accumulator2):
     def compute_metrics(self, accumulator: 'Accumulator'):
         """Computes and returns the result of aggregation."""
         pass
+
+
+class CombinerParams:
+    """Parameters for an combiner.
+
+    Wraps epsilon and delta from the MechanismSpec which are lazily loaded.
```

Comment:
nit: I'm likely missing some context.. why is that lazily loaded, and where is this loading happens?

nit2: Maybe we can stay less technical in the class doc, and add more comments to the code. Maybe we can say here something like "wraps all the information needed by the combiner to do the differentially-private computation, e.g. privacy budget and mechanism.

WDYT?